bin/rcrestore

```diff
@@ -0,0 +1,58 @@
+#! /bin/bash
+
+if [[ ! -f $1 ]]
+then
+    echo "Usage: snap run rocketchat-server.rcrestore $SNAP_USER_COMMON/backup_file.tgz"
+    exit
+fi
+
+cd ${1%/*}
+if [[ -z $(pwd | grep "$SNAP_USER_COMMON") ]]
```

Comment:
Just out of curiosity.  Why change folder and then check pwd?  Why not do something like:
```
if [ ! -f $SNAP_USER_COMMON ]; then
    echo "File not found!"
fi
```